nio/router/diagnostic.py

```diff
@@ -16,7 +16,7 @@ def __init__(self):
 
         self._start_time = None
         self._instance_id = None
-        self._service_name = None
```

Comment:
Can we provide both name and ID in most of these status/diagnostic signals? That would make it backwards compatible still and I think in many cases we would want to use the name too. Having both available would be best.